### ISSUE TITLE
BDV does not work on Ubuntu 15.04 over VNC 

### ISSUE THREAD

**Issue by axtimwalde** [bigdataviewer/bigdataviewer-core#18]
It is may be something simple because I have seen other Swing based applications working (e.g. TrakEM2, File chooser)

```
(Fiji Is Just) ImageJ 2.0.0-rc-44/1.50e; Java 1.8.0_45-internal [64-bit]; Linux 3.19.0-49-generic; 81256MB of 90987MB (89%)
 
java.lang.IllegalArgumentException: invalid range properties
	at javax.swing.DefaultBoundedRangeModel.<init>(DefaultBoundedRangeModel.java:102)
	at javax.swing.JSlider.<init>(JSlider.java:274)
	at bdv.tools.brightness.SliderPanel.<init>(SliderPanel.java:72)
	at bdv.tools.brightness.BrightnessDialog$MinMaxPanel.<init>(BrightnessDialog.java:343)
	at bdv.tools.brightness.BrightnessDialog$MinMaxPanels.recreateContent(BrightnessDialog.java:273)
	at bdv.tools.brightness.BrightnessDialog$MinMaxPanels.<init>(BrightnessDialog.java:258)
	at bdv.tools.brightness.BrightnessDialog.<init>(BrightnessDialog.java:87)
	at bdv.BigDataViewer.<init>(BigDataViewer.java:357)
	at bdv.BigDataViewer.<init>(BigDataViewer.java:687)
	at bdv.BigDataViewer.<init>(BigDataViewer.java:643)
	at bdv.ij.OpenImagePlusPlugIn.run(OpenImagePlusPlugIn.java:177)
	at ij.IJ.runUserPlugIn(IJ.java:212)
	at ij.IJ.runPlugIn(IJ.java:176)
	at ij.Executer.runCommand(Executer.java:136)
	at ij.Executer.run(Executer.java:65)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by tpietzsch** [comment]
@axtimwalde This is not VNC-related most likely. I think it is related to incorrect rounding in the brightness sliders when the data is float/double. I also ran into it last week. I'll look into it.

**Comment by tpietzsch** [comment]
@axtimwalde it should be fixed by this 30f2832ef36abd423f9cf4577161b2abe0de388b
Can you verify?

**Comment by ctrueden** [comment]
Closing for tidiness. Reopen if not fixed.
